proto/pulumi/resource.proto

```diff
@@ -238,3 +240,9 @@ message TransformResponse {
     google.protobuf.Struct properties = 1; // the transformed input properties.
     TransformResourceOptions options = 2; // the options for the resource.
 }
+
+message RegisterDefaultProviderRequest {
+    string provider = 1; // the provider to register as the default provider.
```

Comment:
Is `provider` a urn, can the comment say that?

Reply:
It should be a 'ref' of the form "<URN>::<ID>"